krossbow-stomp-core/src/commonMain/kotlin/org/hildan/krossbow/stomp/StompClient.kt

```diff
@@ -40,10 +40,10 @@ class StompClient(
      * @throws ConnectionTimeout if this method takes longer than the configured
      * [timeout][StompConfig.connectionTimeoutMillis] (as a whole for both WS connect and STOMP connect)
      */
-    suspend fun connect(url: String, login: String? = null, passcode: String? = null): StompSession {
+    suspend fun connect(url: String, login: String? = null, passcode: String? = null, customHeaders: Map<String, String> = emptyMap()): StompSession {
```

Comment:
Now that I think of it, I might add custom headers for the websocket connection too later.
To avoid a backwards-incompatible change in the future, could you please rename this param `customStompConnectHeaders`.
Also please add a relevant sentence in the doc to explain what the param does.

Reply:
Thank you for your answer. 
Learning every day :-1:

Reply:
Well thank *you* for the contribution! 😄